src/lib/products.ts

```diff
@@ -55,9 +55,9 @@ function isProductSlug(string: string): string is ProductSlug {
 const productSlugs = Object.keys(productSlugsToNames) as ProductSlug[]
 
 /**
- * An array of product slugs which are "active" on the site. Currently all but sentinel.
+ * An array of product slugs which are "active" on the site. Currently all products.
  */
-const activeProductSlugs = productSlugs.filter((slug) => slug !== 'sentinel')
```

Comment:
You can remove `activeProductSlugs` entirely from this file (including from the exports on L71) as all products are reflected within `productSlugs` now